app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ErrorsController < ApplicationController
+  def file_not_found
```

Comment:
these methods need to return the appropriate HTTP status code, otherwise the error pages are rendered with `200 OK` which is really confusing